features/bootstrap/FeatureContext.php

```diff
@@ -1,7 +1,41 @@
 <?php
 
-include_once(__DIR__ . '/../../php/class-ee.php');
-include_once(__DIR__ . '/../../php/utils.php');
+define('EE_ROOT',__DIR__ . '/../..');
+
+include_once(EE_ROOT . '/php/class-ee.php');
```

Comment:
same for this, use one type of include_once. Here you've used brackets and not for other places.
i.e: https://github.com/EasyEngine/easyengine/pull/1140/files#diff-8a50a8f3567ab9a03107fd86eeaf827eR28

Also check `require_once`.